orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AuditService.java

```diff
@@ -86,6 +89,12 @@ public void submitAuditEvent(
                                         .orElse("UNKNOWN"))
                         .withUser(user);
 
+        if (ThreadContext.get(TXMA_ENCODED_HEADER.getFieldName()) != null) {
+            txmaAuditEvent.addRestricted(
+                    "device_information",
```

Comment:
Are we ever sending anything in `device_information` at the moment? This will overwrite I think, not merge